### PR TITLE
Prebid 9: Removing innerText & adding eslint rule

### PR DESCRIPTION
**Pull request by mkomorski**
## Type of change

- [ ] Bugfix
- [x] Feature
- [ ] New bidder adapter  
- [ ] Updated bidder adapter   
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes

- [ ] Does this change affect user-facing APIs or examples documented on http://prebid.org?
- [ ] Other

## Other information

#11233

### CODE REVIEW

File: modules/freewheel-sspBidAdapter.js [hunk 1]
```diff
@@ -76,7 +76,7 @@ function getPricing(xmlNode) {
     var priceNode = pricingExtNode.querySelector('Price');
     princingData = {
       currency: priceNode.getAttribute('currency'),
-      price: priceNode.textContent || priceNode.innerText
```
Comment (ChrisHuie) [inline modules/freewheel-sspBidAdapter.js:79]: You can remove freewhell from this pr because #11534

Comment (mkomorski) [reply modules/freewheel-sspBidAdapter.js]: done

**Comment by patmmccann** [comment]
would these tests pass with innerHTML instead of textContent

**Comment by patmmccann** [comment]
LGTM

**Pull request merged**